.vscode/launch.json

```diff
@@ -0,0 +1,33 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+      {
+        "name": "ng serve",
+        "type": "chrome",
+        "request": "launch",
+        "url": "http://localhost:4200/#",
+        "webRoot": "${workspaceFolder}",
+        "sourceMapPathOverrides": {
+          "webpack:/./*": "${webRoot}/*",
+          "webpack:/src/*": "${webRoot}/src/*",
+          "webpack:/*": "*",
+          "webpack:/./~/*": "${webRoot}/node_modules/*"
+        }
+      },
+      {
+        "name": "ng test",
+        "type": "chrome",
+        "request": "launch",
+        "url": "http://localhost:9876/debug.html",
+        "webRoot": "${workspaceFolder}"
+      },
+      {
+        "name": "ng e2e",
+        "type": "node",
+        "request": "launch",
+        "program": "${workspaceFolder}/node_modules/protractor/bin/protractor",
+        "protocol": "inspector",
+        "args": ["${workspaceFolder}/protractor.conf.js"]
+      }
+    ]
+  }
```

Comment:
Remove this file

Reply:
Sorry :)

Reply:
Actually it's not really your fault, it's in the gitignore, can you remove exceptions here:

https://github.com/K-Fet/K-App/blob/2a147461744dce36b5d046cc1f0e328f3da7c0a6/.gitignore#L26-L30

Reply:
Implemented